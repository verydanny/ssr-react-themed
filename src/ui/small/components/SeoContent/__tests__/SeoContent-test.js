import React from 'react'
import { shallow } from 'enzyme'
import WrappedSeoContent from '../SeoContent'

const SeoContent = WrappedSeoContent.WrappedComponent

describe('SeoContent', () => {
  const props = {
    apartmentLinks: {},
    propertyTypeLinks: {},
    cityLinks: {},
    neighborhoods: {},
  }

  it('displays nearby colleges if shouldDisplayNearbyColleges is true', () => {
    const wrapper = shallow(<SeoContent {...props} shouldDisplayNearbyColleges />)
    expect(wrapper).toMatchSnapshot()
  })

  it('does not display nearby colleges if shouldDisplayNearbyColleges is false', () => {
    const wrapper = shallow(<SeoContent {...props} shouldDisplayNearbyColleges={false} />)
    expect(wrapper).toMatchSnapshot()
  })
})
