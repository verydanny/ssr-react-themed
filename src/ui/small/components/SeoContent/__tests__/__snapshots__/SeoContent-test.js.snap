// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SeoContent displays nearby colleges if shouldDisplayNearbyColleges is true 1`] = `
<div
  data-tid="seo-content"
>
  <Connect(Themed(LinkList))
    header=" Apartment Options"
  />
  <Connect(Themed(LinkList))
    header="Search by Property Type"
  />
  <Connect(Themed(DfpAdSlot))
    adKey="gpt_list_srp_mobile_seocontent"
    tid="ad-dfp-listview"
  />
  <Themed(Nearby)
    header="Find undefined in Nearby Cities"
    linkTagItem="by_nearby_cities"
    linkTagSection="local_info"
  />
  <Themed(Nearby)
    header="Find undefined in Nearby Neighborhoods"
    linkTagItem="by_nearby_neighborhoods"
    linkTagSection="local_info"
  />
  <Themed(Nearby)
    header="Find undefined Near Local Colleges"
    linkTagItem="by_nearby_colleges"
    linkTagSection="local_info"
    maxLinks={10}
  />
  <Themed(Nearby)
    header="Find undefined Near Local Military Bases"
    linkTagItem="by_nearby_military"
    linkTagSection="local_info"
  />
  <Connect(Themed(DfpAdSlot))
    adKey="gpt_list_srp_mobile_footer"
    key="gpt_list_srp_mobile_footer"
    tid="ad-bottom-list"
  />
</div>
`;

exports[`SeoContent does not display nearby colleges if shouldDisplayNearbyColleges is false 1`] = `
<div
  data-tid="seo-content"
>
  <Connect(Themed(LinkList))
    header=" Apartment Options"
  />
  <Connect(Themed(LinkList))
    header="Search by Property Type"
  />
  <Connect(Themed(DfpAdSlot))
    adKey="gpt_list_srp_mobile_seocontent"
    tid="ad-dfp-listview"
  />
  <Themed(Nearby)
    header="Find undefined in Nearby Cities"
    linkTagItem="by_nearby_cities"
    linkTagSection="local_info"
  />
  <Themed(Nearby)
    header="Find undefined in Nearby Neighborhoods"
    linkTagItem="by_nearby_neighborhoods"
    linkTagSection="local_info"
  />
  <Themed(Nearby)
    header="Find undefined Near Local Military Bases"
    linkTagItem="by_nearby_military"
    linkTagSection="local_info"
  />
  <Connect(Themed(DfpAdSlot))
    adKey="gpt_list_srp_mobile_footer"
    key="gpt_list_srp_mobile_footer"
    tid="ad-bottom-list"
  />
</div>
`;
